package.json

```diff
@@ -38,17 +38,18 @@
   },
   "dependencies": {
     "@govflanders/vl-ui-form-validation": "^3.9.1",
-    "vl-ui-core": "^5.5.5"
+    "vl-ui-core": "milieuinfo/webcomponent-vl-ui-core#mixin"
   },
   "peerDependencies": {
-    "vl-ui-core": "^5.5.5"
+    "vl-ui-core": "milieuinfo/webcomponent-vl-ui-core#mixin"
```

Comment:
vergeet niet aan te passen naar een nieuwe core release versie

Reply:
Nee, staat bij TODO in description.